main/tclx/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Shannon Noe <snoe925@gmail.com>
+# Maintainer: Shannon Noe <snoe925@gmail.com>
+pkgname=tclx
+pkgver=8.4.1
+pkgrel=0
+pkgdesc="TclX extension to Tcl"
+url="http://tclx.sourceforge.net/"
+arch="all"
+license="BSD"
+depends="tcl"
+depends_dev="tcl-dev"
+makedepends="tcl-dev"
+subpackages="$pkgname-doc $pkgname-dev"
+source="http://downloads.sourceforge.net/project/tclx/TclX/$pkgver/$pkgname$pkgver.tar.bz2
+	"
+
+_major=${pkgver%.*}
+_builddir="$srcdir"/tclx${_major}
+
+build() {
+	local _64bit="--disable-64bit"
+	case "$CARCH" in
+		x86_64) _64bit="--enable-64bit";;
```

Comment:
We normally prefer to indent case clauses at same level with the case. Though, this type of indentation has been on some other aports too.